crates/bevy_pbr/src/render/mesh_view_types.wgsl

```diff
@@ -10,6 +10,8 @@ struct View {
     world_position: vec3<f32>,
     width: f32,
     height: f32,
+    viewport_x: f32,
+    viewport_y: f32,
```

Comment:
Are the width and height the viewport width and height? If so, maybe merge the four f32s into a vec4 and put it before the world_position.

Reply:
I see from where the uniform data is created, they are indeed the viewport width and height.

Reply:
can do, just didn't want to cause breaking changes for a bugfix?  i assume people rely on height and width in their shaders

Reply:
We’re not really trying to avoid breaking changes at this point, as far as I am aware. We just need to document them. Ideally in the PR description template format. :)